solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -521,6 +541,94 @@ describe("RandomBeacon - Relay", () => {
     })
   })
 
+  describe("reportUnauthorizedSigning", () => {
+    beforeEach(async () => {
+      await createGroup(randomBeacon as RandomBeaconStub, signers)
+
+      await (randomBeacon as RandomBeaconStub).setMinimumStake(to1e18(1))
+
+      await approveTestToken()
+      await randomBeacon.connect(requester).requestRelayEntry(ZERO_ADDRESS)
+    })
```

Comment:
Can we put an empty line after `beforeEach`?

Reply:
done